sherlock_project/sherlock.py

```diff
@@ -167,14 +167,14 @@ def multiple_usernames(username):
 
 
 def sherlock(
-    username,
-    site_data,
+    username: str,
+    site_data: dict,
     query_notify: QueryNotify,
     tor: bool = False,
     unique_tor: bool = False,
     dump_response: bool = False,
-    proxy=None,
-    timeout=60,
+    proxy: str = None,
```

Comment:
```suggestion
    proxy: Optional[str] = None,
```
`from typing import Optional`
Since it defaults to None, `Optional` can be better here

Reply:
Agh. Good catch. Thanks @GuardianWang